### PR TITLE
chore: introduce simple issue templates

### PR DESCRIPTION
**Pull request by alexeagle**
The immediate goal is to ensure more uniformity of the module wanted requests, and to collect links to upstream issues. Hopefully this automates a bit of the maintenance work for this repo.

### CODE REVIEW

**Comment by alexeagle** [comment]
Note, after studying the problem a bit more, I think that as long as we make the search page look okay:
https://github.com/bazelbuild/bazel-central-registry/issues?q=is%3Aissue+is%3Aopen+label%3A%22module+wanted%22
this is good enough to track the modules we need to add, rather than maintaining one tracking issue like I proposed in the meeting today.

@hobofan FYI since you +1 that suggestion, any concern with doing it this way?

File: .github/ISSUE_TEMPLATE/module_wanted.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+name: Module Wanted
+description: Request that a module be added to the registry
+title: "wanted: [github path of the module, e.g. bazelbuild/rules_foo]"
+labels: ["module wanted"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        The Central Registry is populated by community contributions.
+        You can file an issue to request that a volunteer add it to the registry.
```
Comment (fmeum) [inline .github/ISSUE_TEMPLATE/module_wanted.yaml:10]: "request" sounds a bit strong here, even when combined with the following sentence.

**Review by fmeum — APPROVED** [review]

**Review by meteorcloudy — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/module_wanted.yaml (hunk 1 above)
Comment (alexeagle) [reply .github/ISSUE_TEMPLATE/module_wanted.yaml]: changed to

> You can file an issue to indicate that you depend on a module which isn't available here.
>        A volunteer may contribute that module, but there is no guarantee.
>        You may want to send a PR yourself, or consider sponsoring the work.

**Review by fmeum — APPROVED** [review]

**Comment by alexeagle** [comment]
hm interesting, did we mean to require a re-review of changes to this repo? I think the "LGTM with comments" pattern is more productive. @meteorcloudy IMO we should turn that setting off, and instead require that conversations are resolved.

**Comment by meteorcloudy** [comment]
Oh, I think it's due to
image

**Comment by meteorcloudy** [comment]
OK, I disabled that once and enabled:
image

**Comment by alexeagle** [comment]
Thank you!

**Pull request merged**